Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md

```diff
@@ -0,0 +1,110 @@
+# Upgrading python interpreter version from 3.7.10 to 3.8.12
+
+Currently, in Oppia we are utilizing python interpreter version 3.7.10. We are now in a phase of switching to a more updated interpreter version 3.8.12. For effective and effortless switching to the newer version, this guide will provide a brief and detailed instruction on how to achieve just that. Please follow the below mentioned instructions carefully.
```

Comment:
I think "Currently" will go out of date quite soon. Maybe change this to something like: "Oppia changed the Python version it uses from 3.7.10 to 3.8.12 in Oct 2022. This page is aimed at developers who are currently using Python 3.7.10 and need to upgrade to 3.8.12."

Then you can have a new paragraph starting "For effective and effortless..." etc.

Reply:
done